components/MarkdownTextBlock/src/Renderers/UWPRenderer.cs

```diff
@@ -0,0 +1,157 @@
+using CommunityToolkit.Labs.WinUI.MarkdownTextBlock.Renderers.ObjectRenderers;
+using CommunityToolkit.Labs.WinUI.MarkdownTextBlock.Renderers.ObjectRenderers.Inlines;
+using CommunityToolkit.Labs.WinUI.MarkdownTextBlock.Renderers.ObjectRenderers.Extensions;
+using CommunityToolkit.Labs.WinUI.MarkdownTextBlock.TextElements;
+using Markdig.Renderers;
+using Markdig.Syntax;
+using Markdig.Helpers;
+
+namespace CommunityToolkit.Labs.WinUI.MarkdownTextBlock.Renderers;
+
+public class UWPRenderer : RendererBase
```

Comment:
Wrapping head around all the connections still with this, but we probably at minimum would want to call this a 'WinUIRenderer' if not something else?

Reply:
Addressed in latest version